scanner/updater/export.go

```diff
@@ -62,30 +67,44 @@ func Export(ctx context.Context, outputDir string) error {
 	outOfTree = append(outOfTree, updaterSet.Updaters())
 
 	for i, uSet := range [][]string{
-		{"oracle", "photon", "suse", "aws", "rhcc"},
-		{"alpine", "rhel", "ubuntu", "osv", "debian"},
+		{"oracle"}, {"photon"},
```

Comment:
Following @jvdm's suggestion, we download from one single data source at a time. This makes sure the jsonblob and zstd writer utilize memory more efficiently.